.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Report a bug.
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+\-------------------------------------------------------------------------------------------------
+**IS THIS BUG RELATED TO EXTERNAL TOOLS? (e.g. git, less, etc.)**
+
+If it is, there's a good chance we might need to know a little bit more about the environment
+which you're running `bat` on.
+
+**If you're on Windows:**  
+Please tell us about your:
+
+- Windows Version
+
+**If you're on Linux or MacOS:**  
+Please run the script at https://github.com/sharkdp/bat/blob/diag-tools/diagnostics/info.sh and
+paste the output at the bottom of the bug report.
+
+**Once you're done:**  
+Please remove this header and continue.
+\-------------------------------------------------------------------------------------------------
+
+**What do you expect to happen?**  
```

Comment:
Typically, I see these two points the other way around:
1. Describe your question or bug.
2. What did you expect to happen.

Reply:
Fixed.